documentation/src/main/asciidoc/topics/rest_api_v2.adoc

```diff
@@ -94,6 +94,25 @@ include::rest_examples/get_v2_configuration.adoc[]
 |Sets the required format to return content. Supported formats are `application/xml` and `application/json`. The default is `application/json`. See link:#rest_accept[Accept] for more information.
 |===
 
+[[rest_v2_cache_detail]]
+=== Retrieving All the details related to a cache
+
+To retrieve the details of a {brandname} cache, invoke a `GET` request:
+
+[source,options="nowrap",subs=attributes+]
+----
+include::rest_examples/get_v2_all_details.adoc[]
+----
+
+.Headers
+|===
+|Header |Required or Optional |Parameter
+
+|`Accept`
+|OPTIONAL
+|Sets the required format to return content. Supported formats are `application/xml` and `application/json`. The default is `application/json`. See link:#rest_accept[Accept] for more information.
+|===
+
```

Comment:
The doc is missing some info:
* Example output
* Table describing the fields

Reply:
there are too many outputs ... but I will describe